routes/licenses.js

```diff
@@ -10,8 +10,21 @@ const mockResponse = [
 routes.push({
   path: '/licenses',
   method: 'GET',
-  options: {},
+  options: {
+    description: 'Licenses Index',
+    notes: 'Returns a List of all Licenses',
+  },
   handler: async (request, h) => h.response(mockResponse),
 });
 
+routes.push({
+  path: '/license/{file}',
+  method: 'GET',
+  options: {
+    description: 'Image License',
```

Comment:
I'm not 100% sure I already got everything the `Lizenzhinweisgenerator` is doing, but from my current understanding, I though it would generate licenses for any type of media files, not necessary only images 🤔

Reply:
Not sure about that, too 😄
I just wanted to stick to our use case for now; the `API Endpoints` document refers to the file-param as `image`, so I adopted that.